api/controllers/comment.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+
+const config = require('../config');
+const db = require('../models');
+const common = require('../common');
+
+
+/**
+ * @apiGroup Comment
+ * @api {get} /comment/project/:projectid  Get Comment List of Particular Project
+ * @apiHeader {String} authorization 
+ *                              A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object}            List of Comments 
+ */
+
+router.get('/project/:id', common.jwt(), (req, res, next) =>{
+    if(!req.params.id) return next("project id not defined");
+    let projectID = req.params.id;
+    let limit = req.query.limit || 100;
+    db.Projects.findById(projectID, async (err, project)=>{
+        if(err) return next(err);
+        if(!project) return res.status(404).end();
+        if(!common.ismember(req.user, project)) return res.status(401).end("you are not a member of this project");
```

Comment:
We should also let admins / guests to be able to view comments. For public project, we can also let any logged in users to view comments.